src/relation_info.c

```diff
@@ -411,7 +414,7 @@ build_pathman_relation_info(Oid relid, Datum *values)
 
 	/* Create a new memory context to store expression tree etc */
 	prel_mcxt = AllocSetContextCreate(PathmanParentsCacheContext,
-									  __FUNCTION__,
```

Comment:
Why?

Reply:
Because since 9fa6f00b1308dd10da4eca2f31ccbfc7b35bb461 `AllocSetContextCreate` ensures that this arg is `__builtin_constant_p`, and  `__FUNCTION__` stores func name in global buffer.

Reply:
Ok, thanks!